cli/src/main/java/org/apache/falcon/cli/FalconMetadataCLI.java

```diff
@@ -196,63 +232,9 @@ public void metadataCommand(CommandLine commandLine, FalconClient client) {
         OUT.get().println(result);
     }
 
-    private void validatePipelineName(String pipeline) {
+    public static void validatePipelineName(String pipeline) {
```

Comment:
I know this method is not added as part of this JIRA but can we replace it with validateNotEmpty?